devops/providers/azure-devops/templates/azure-pipelines.yml

```diff
@@ -48,6 +48,10 @@ variables:
   value: 'xterm-color'
 - name: TF_WARN_OUTPUT_ERRORS
   value: 1
+- name: STATIC_ASE_NAME
```

Comment:
Should we move this into the pipeline variable settings within the azure devops portal as opposed to defining it within our build definition?

Reply:
Honestly I'd love some guidance on this one -- this choice has always seemed pretty arbitrary to me. I suppose the pros of putting it into AZDO is that other users of this could define it differently.

Aside from this, is there guidance for deciding where to put these variables?

Reply:
I converted these to pipeline variables.